decidim-elections/db/migrate/20220711112802_rename_dataset_file_to_filename.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RenameDatasetFileToFilename < ActiveRecord::Migration[6.1]
```

Comment:
I left a comment in the other related thread but I believe you can get the filename through the blob object.